backend/src/personnel/personnel.service.ts

```diff
@@ -77,23 +77,40 @@ export class PersonnelService {
    * @param id
    * @returns
    */
-  async findOne(id: string): Promise<PersonnelEntity> {
-    const person = await this.personnelRepository.findOneOrFail({
-      where: { id },
-      relations: [
-        'certifications',
-        'certifications.certification',
-        'tools',
-        'tools.tool',
-        'homeLocation',
-        'workLocation',
-        'recommitment',
-        'recommitment.recommitmentCycle',
-        'languages',
-      ],
-    });
-
-    return person;
+  async findOne(id?: string, email?: string): Promise<PersonnelEntity> {
```

Comment:
Might suggest instead separating this to findOneById and findOneByEmail. This function could technically have nothing passed in and return void